tests/requirements-307.txt

```diff
@@ -1,26 +1,26 @@
 aiofiles>=0.5.0
-aiohttp>=3.7.4
+aiohttp>=3.8.3
 boto3>=1.17.74
-celery>=5.0.5
-# TODO: Remove this when the fix is available in non beta
-# We have to use a beta version of kombu on Python 3.7
-# because only that fixes "AttributeError: 'EntryPoints' object has no attribute 'get'"
-# that we have in the CI: https://app.circleci.com/pipelines/github/instana/python-sensor/1372/workflows/90878561-aada-49f8-8a1b-78562aa05aab/jobs/7478
-# the issue: https://github.com/celery/kombu/issues/1600
-# the PR: https://github.com/celery/kombu/pull/1601/files
-# the release notes: https://github.com/celery/kombu/releases/tag/v5.3.0b2
-kombu>=5.3.0b2
+
+# TODO: importlib_metadata package removed deprecated entry_point interfaces on 
+# version 5.0.0 [1], and this change impacts celery >= 5.2.7 [2] running with 
+# python 3.7.X (it doesn't impact >= 3.8). For this reason, we control celery 
+# and importlib_metadata versions on python 3.7 environments.
+# [1] https://github.com/python/importlib_metadata/pull/405
+# [2] https://github.com/celery/celery/issues/7783
+celery>=4.3.0,<=5.2.7
```

Comment:
I think this line could be `celery>=5.2.7` like in the rest of the requirements.